lib/App/Rakubrew/Build.pm

```diff
@@ -187,16 +187,25 @@ sub build_triple {
 
 sub build_zef {
     my $version = shift;
+    my $zef_version = shift;
 
     _check_git();
-
     chdir catdir($versions_dir, $version);
     unless (-d 'zef') {
         run "$GIT clone $git_repos{zef}";
     }
     chdir 'zef';
-    run "$GIT pull -q";
-    run "$GIT checkout";
+    my %tags = map  { chomp($_); $_ => 1 } `$GIT tag`;
+    unless ( $zef_version && $tags{$zef_version} ) {
```

Comment:
Maybe `if ( $zef_version && !$tags{$zef_version} ) { say STDERR "Couldn't find version $zef_version, aborting."; exit 1; }`

Reply:
Instead of trying to trying to install a default one? OK

Reply:
Used `die` which does the same.

Reply:
The reason I use `say` and `exit 1` throughout the code is that the output looks cleaner as it doesn't include position markers / stacktrace. Can you change that? Otherwise the PR looks good. Thank you for your effort and not giving up!

Reply:
You simply need to add `\n` at the end of the message, as I have done (from [here](https://stackoverflow.com/questions/2022039/how-can-i-output-a-die-message-without-the-location-information). `die` sends the message more clearly for the programmer, I guess.

Reply:
I didn't know that!

Reply:
I remembered it vaguely, but I had to look it up. Not so used to program Perl lately :-)